src/providers/ipa/ipa_subdomains_server.c

```diff
@@ -1195,7 +1195,8 @@ int ipa_ad_subdom_init(struct be_ctx *be_ctx,
         /* We need to handle keytabs created by IPA oddjob script gracefully
          * even if we're running as root and IPA creates them as the SSSD user
          */
-        ret = sss_user_by_name_or_uid(SSSD_USER,
+        ret = sss_user_by_name_or_uid(&id_ctx->ops,
```

Comment:
Can't we simply use `sss_sssd_user_uid_and_gid()` instead?

Reply:
I haven't changed this to have the smallest changes in the patch.